tools/analysis/analysis_runner.py

```diff
@@ -0,0 +1,77 @@
+# Analysis runner.
+import argparse
+import os
+import subprocess
+
+# TODO: Support customizing these.
+_TOPIC = 'projects/ossf-malware-analysis/topics/workers'
+_PACKAGES_BUCKET = 'ossf-malware-analysis-packages'
+
+
+def _upload_file(local_path):
+  # TODO: figure out a better way to key these.
+  filename = os.path.basename(local_path)
+  upload_path = f'gs://{_PACKAGES_BUCKET}/{filename}'
+  print('Uploading', local_path, 'to', upload_path)
+  subprocess.run(
+    ('gsutil', 'cp', local_path, upload_path), check=True)
+
+  return filename
+
+
+def _request(name, ecosystem, version, local_file=None, results_bucket=None):
+  attributes = [
+    'name=' + name,
+    'ecosystem=' + ecosystem,
+  ]
+
+  if version:
+    attributes.append('version=' + version)
+
+  if local_file:
+    uploaded_path = _upload_file(local_file)
+    attributes.append('package_path=' + uploaded_path)
+
+  if results_bucket:
+    attributes.append('results_bucket_override=' + results_bucket)
+
+  print('Requesting analysis with', attributes)
+  subprocess.run(
+    ('gcloud', 'pubsub', 'topics', 'publish',
```

Comment:
As above. Might be worth listing `gcloud` as a dep.

Reply:
This is all part of Google Cloud SDK. I've added this to the README.